prescient/engine/egret/egret_plugin.py

```diff
@@ -675,6 +679,38 @@ def _ensure_reserve_factor_honored(options:Options, md:EgretModel, time_periods:
             if reserve_reqs[t] < min_reserve:
                 reserve_reqs[t] = min_reserve
 
+def _ensure_contingencies_monitored(options:Options, md:EgretModel, initial_day:bool = False) -> None:
+    ''' Add contingency screening, if that option is enabled '''
+    if initial_day:
+        _ensure_contingencies_monitored.contingency_dicts = {}
```

Comment:
Use of this function-local static variable makes the code not thread safe. For now that's OK, but at some point we might want to address it. create_sced_instance.shutdown_curves is another example of this.  

Linked to Issue #130.